src/dispatch/case/service.py

```diff
@@ -80,6 +83,16 @@ def get_all_by_status(*, db_session, project_id: int, status: str) -> List[Optio
     )
 
 
+def get_count_of_case_types(*, db_session) -> list[tuple[str, int]]:
```

Comment:
should be `case/type` dir

Reply:
Could we generalize and parameterize this function to accept the objects for which we want the counts, so we don't have to write one for each data type?

Reply:
Yeah potentially, it may get tricky if we need to join across tables.